CODEOWNERS

```diff
@@ -1,46 +1,97 @@
-# Live DX Team
-.github/              @ledgerhq/live-eng-core
-scripts/              @ledgerhq/live-eng-core
-tools/                @ledgerhq/live-eng-core
-fastlane/             @ledgerhq/live-eng-core
+# How does it work? => https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies JS files, only @js-owner and not the global
+# owner(s) will be requested for a review.
 
-# Live Blockchain Support Team
-apps/**/families/                                                    @ledgerhq/live-blockchain-support
-apps/ledger-live-desktop/src/live-common-set-supported-currencies.*  @ledgerhq/live-blockchain-support
-apps/ledger-live-mobile/src/live-common-setup.*                      @ledgerhq/live-blockchain-support
-libs/ledger-live-common/src/families                                 @ledgerhq/live-blockchain-support
-libs/ledgerjs/packages/cryptoassets                                  @ledgerhq/live-blockchain-support
-libs/ledgerjs/packages/types-cryptoassets                            @ledgerhq/live-blockchain-support
-libs/ledgerjs/packages/hw-app-*                                      @ledgerhq/live-blockchain-support
-libs/coin-*                                                          @ledgerhq/live-blockchain-support
-libs/domain-service                                                  @ledgerhq/live-blockchain-support
-ledgerjs/packages/evm-tools                                          @ledgerhq/live-blockchain-support
-docs                                                                 @ledgerhq/live-blockchain-support
+# Hub team
+## by default, hub manage apps and libs related to it & then we overrides many exceptions
+apps/ledger-live-desktop/                                @ledgerhq/live-hub
+apps/ledger-live-mobile/                                 @ledgerhq/live-hub
+libs/**/errors/                                          @ledgerhq/live-hub
+libs/**/logs/                                            @ledgerhq/live-hub
+libs/**/types-live/                                      @ledgerhq/live-hub
+libs/env/                                                @ledgerhq/live-hub
```

Comment:
These ones belong a bit to everyone, shouldn't we put them in all teams?

Reply:
if it belongs to everyone, it belongs to no one, i think the idea for now is to settle a default owner for each path. if something is too "cross team" we may prefer to unset completely from any ownership (PR without auto "reviewers")